packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -150,13 +159,18 @@ export class UnfurlService {
                 element = await page.$('body');
             }
 
-            if (lightdashPage === LightdashPage.DASHBOARD) {
-                // Remove navbar from screenshot
+            const removeComponents = async (selectorToRemove: string) => {
                 await page.evaluate((sel: any) => {
                     // @ts-ignore
                     const elements = document.querySelectorAll(sel);
                     elements.forEach((el) => el.parentNode.removeChild(el));
-                }, '.bp4-navbar');
+                }, selectorToRemove);
+            };
+            if (lightdashPage === LightdashPage.DASHBOARD) {
+                // Remove navbar from screenshot
+                removeComponents('.bp4-navbar');
+                removeComponents('button');
```

Comment:
Remove buttons and filter icon , not really sure if removing useless content s will make screenshooting faster

![Screenshot from 2023-02-24 09-52-34](https://user-images.githubusercontent.com/1983672/221135481-99af3374-8956-4a48-9ebd-7dbfc1f739da.png)